.github/workflows/tag.yml

```diff
@@ -68,7 +68,11 @@ jobs:
         with:
           ref: ${{ inputs.ref }}
           token: ${{ secrets.BROADBOT_TOKEN }} # this allows the push to succeed later
-          persist-credentials: false
```

Comment:
Removing this makes sense because we need the git credentials when we go to tag the branch. I think this is the only needed change. 

The previous run appeared to fail because the tag wasn't created. So, then the build action couldn't check out the non-existent tag.